app/Controllers/SearchController.php

```
@@ -199,6 +199,9 @@ public function unifiedSearch(Request $request, Response $response, mysqli $db):
             $publisherResults = $this->searchPublishers($db, $q);
             $results = array_merge($results, $publisherResults);
 
+            // Allow active plugins (e.g. Archives) to append extra result sets.
+            $results = \App\Support\Hooks::apply('search.unified.sources', $results, [$q]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\Hooks' in method 'unifiedSearch'. (undefined)

(StaticAccess)